src/components/NotificationsModal/Notification.tsx

```diff
@@ -0,0 +1,243 @@
+import { useRouter } from 'next/router';
```

Comment:
responsible for rendering any kind of notification here 
<img width="419" alt="image" src="https://user-images.githubusercontent.com/6754223/200647649-3cbca8d2-a9fe-4350-89d1-95c65f125e96.png">